cmd/internal/find/types/options.go

```diff
@@ -54,11 +57,41 @@ func (opts *Options) MarkAsSet(flag string) {
 func (opts *Options) FlagSet() *flag.FlagSet {
 	// Create the flag-set that's tied to our options.
 	fs := flag.NewFlagSet("options", flag.ContinueOnError)
-	// TODO: Handle usage later
-	fs.Usage = func() {}
 	fs.SetOutput(ioutil.Discard)
 	fs.BoolVar(&opts.Depth, DepthFlag, opts.Depth, "")
 	fs.UintVar(&opts.Mindepth, MindepthFlag, opts.Mindepth, "")
 	fs.IntVar(&opts.Maxdepth, MaxdepthFlag, opts.Maxdepth, "")
 	return fs
 }
+
+// OptionsTable returns a table containing all of `wash find`'s available
+// options
+func OptionsTable() *cmdutil.Table {
+	return cmdutil.NewTable(
+		[]string{"Flags:",                 ""},
+		[]string{"      -depth",           "Visit the children first before the parent (default false)"},
+		[]string{"      -mindepth levels", "Do not print entries at levels less than levels (default 0)"},
+		[]string{"      -maxdepth levels", "Do not print entries at levels greater than levels (default infinity)"},
+		[]string{"  -h, -help",            "Print the usage"},
+		[]string{"  -h, -help <primary>",  "Print a detailed description of the specified primary"},
```

Comment:
This would probably benefit from an example, since it seems to want the primary name without hyphen.